scala-package/core/src/main/scala/org/apache/mxnet/javaapi/Image.scala

```diff
@@ -111,4 +112,21 @@ object Image {
   def toImage(src: NDArray): BufferedImage = {
     org.apache.mxnet.Image.toImage(src)
   }
+
+  /**
+    * Draw bounding boxes on the image
+    * @param src        buffered image to draw on
+    * @param coordinate Contains Map of xmin, xmax, ymin, ymax
+    *                   corresponding to top-left and down-right points
+    * @param names      The name set of the bounding box
+    */
+  def drawBoundingBox(src: BufferedImage,
+                      coordinate: java.util.List[
+                        java.util.Map[java.lang.String, java.lang.Integer]],
+                      names: java.util.List[java.lang.String]): Unit = {
+    val coord = coordinate.asScala.map(
+      ele => ele.asScala.map(ele => (ele._1, Integer2int(ele._2))).toMap).toArray
```

Comment:
It's a bit confusing if you use the same lambda name `ele` in both the inner and outer lambda. Also, it might be cleaner to use the case map `_.asScala.map { case (name, value) => (name, Integer2int(value)) }.toMap).toArray`